gRPC-C++.podspec

```diff
@@ -23,15 +23,15 @@
 Pod::Spec.new do |s|
   s.name     = 'gRPC-C++'
   # TODO (mxyan): use version that match gRPC version when pod is stabilized
-  # version = '1.19.0-dev'
+  # version = '1.20.0-dev'
   version = '0.0.8-dev'
```

Comment:
Should this "-dev" suffix also be removed similar to core-version ?

Reply:
Good point. I am not sure when this version is bumped up. Will leave it to @muxi  to decide.